process.c

```diff
@@ -8189,9 +8224,7 @@ rb_clock_gettime(int argc, VALUE *argv, VALUE _)
 
     VALUE unit = (rb_check_arity(argc, 1, 2) == 2) ? argv[1] : Qnil;
     VALUE clk_id = argv[0];
-#ifdef HAVE_CLOCK_GETTIME
     clockid_t c;
-#endif
```

Comment:
Is there a reason why this code was changed?

Reply:
Absolutely no idea how this happened (but I should have seen it in the diffs).